readthedocs-theme/templates/readthedocs/product.html

```diff
@@ -0,0 +1,248 @@
+{% extends "page.html" %}
+
+{% block body_id %}product{% endblock %}
+
+{% block content %}
+
+  <!-- User Segments -->
+  <section class="ui vertical segment inverted padded" id="user-segments">
+
+    <div class="ui container">
+      <div class="ui grid stackable three column">
+
+        <div class="column">
+
+          <div>
+            <h3 class="ui header inverted">Docs as Code</h3>
+            <p>
+              Read the Docs simplifies the entire process of building and deploying developer documentation. With support for Sphinx, Mkdocs, and Jupyter Book, you can integrate your code and user-facing documentation using the same tools. Create beautiful documentation easily with themes, and preview every commit with pull request previews.
+            </p>
+            
+            <div class="ui list inverted relaxed divided">
+
+              <div class="item">
+                <i class="icon check circle green"></i>
+                <div class="content">
+                  <div class="description">Issue Tracking</div>
+                </div>
+              </div>
+              
+              <div class="item">
+                <i class="icon check circle green"></i>
+                <div class="content">
+                  <div class="description">Version Control</div>
```

Comment:
Apply Wikipedia style guide here too:

```suggestion
                  <div class="description">Version control</div>
```